runtime/parser2/expression.go

```diff
@@ -1094,10 +1124,15 @@ func applyExprMetaLeftDenotation(
 		return left, true
 	}
 
+	skipWhitespace := exprLeftDenotationAllowsWhitespaceAfterToken(p.current.Type)
```

Comment:
Would this not be better named `permitWhitespace` or `allowWhitespace`?

Reply:
Actually reading through I follow now! Nvm

Reply:
improved in 78b79c8409a6c9545ac0889d8a538653a9cdb05d